git_day_1.adoc

```diff
@@ -0,0 +1,138 @@
+Git Gud Speaker Outline - Day 1
+-------------------------------
+:hardbreaks:
+
+The Basics
+~~~~~~~~~~
+The goal of this presentation is to give you an understanding of how *local* git works, and will not extensively
+cover the syntax and associated _how to_ around using git. Specific syntax questions can be resolved at the end
+of the presentation or via google.
+
+Understand Your Audience
+~~~~~~~~~~~~~~~~~~~~~~~~
+* Is there anyone that doesn't know git?
+* If you were on a client site, would you be comfortable teaching them about git?
+* What type of team environment did you use git in? Solo? Small Team? Army?
```

Comment:
I'd remove "army" as some _have_ worked with the US Army.

Reply:
I didn't think about it that way, but picturing a bunch of troops in uniform all stressing over git amuses me somewhat. Will fix.